docs/blog/2019-04-04-how-to-build-a-blog-with-wordpress-and-gatsby-part-2/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 'How To Build A Blog with Wordpress and Gatsby.js - Part 2'
+date: 2019-04-04
+author: Tim Smith
+excerpt: "In the last post, you covered setting up WordPress for use with Gatsby. Today you will cover how to pull the data from WordPress into Gatsby and build pages."
+tags:
+  - wordpress
+  - apis
+  - blog
+  - headless cms
+  - react
+canonicalLink: https://www.iamtimsmith.com/blog/how-to-build-a-blog-with-wordpress-and-gatsby-part-2/
+---
+
+> Note: This is part two in a series of posts about Gatsby.js and WordPress. You can find [Part One Here](/blog/2019-04-03-how-to-build-a-blog-with-wordpress-and-gatsby-part-1), and [Part Three Here](/blog/2019-04-05-how-to-build-a-blog-with-wordpress-and-gatsby-part-3).
+
+In the last post, I covered setting up [WordPress for use with Gatsby](/blog/2019-04-03-how-to-build-a-blog-with-wordpress-and-gatsby-part-1). Today I will cover how to pull the data from WordPress into Gatsby and build pages.
+
+I have set up a WordPress site for you to use with the plugins mentioned in the last post as well as some dummy content to use. If you're curious, my favorite lorem generator is [Fillerama](http://fillerama.io/) which offers random content from Futurama, Monty Python, Star Wars, and more. This is where the content came from.
+
+<video autoplay muted loop width="400">
+  <source src="https://media.giphy.com/media/sDcfxFDozb3bO/giphy.mp4">
+</video>
+
+## Gatsby.js starter
+
+One of the nice things about Gatsby is the starter projects. These are available for most applications and setups including WordPress, although you won't be using that today. You will just be using the starter default so you can configure everything yourself.
+
+In the future, it may be beneficial to take a look at the various starters and see if there's one that meets your needs. This can be a huge time saver and gives you the ability to get straight into the development portion without having to worry about configuration.
+
+### Gatsby default starter
+
+Before I get into the code, I want to talk about the [Gatsby starter default](https://github.com/gatsbyjs/gatsby-starter-default). This project is the default (obviously) project that will be built when you create a new gatsby project with the Command Line Interface, or CLI. It comes with a few plugins, but not too much so it's wide open for customization.
+
+Before you dig into the project files, you need to create your project. First you need to install the gatsby-cli tool with the code below:
+
+![Install the Gatsby CLI tool](images/code1.png)
```

Comment:
It would be great if we would use code snippets - like:
````
```shell
yarn global add gatsby-cli
```
````
or
````
```shell
npm install -g gatsby-cli
```
````
It's much easier for users to follow along if they can copy paste commands

(this comment apply to all terminal / code screenshots)

Reply:
Changed images of code over to actual code samples.